### PR TITLE
added jobs system

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: src/Jobs/BackupSite.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    public function handle()
+    {
+        $serverPassword = $this->server->getPassword();
+
+        $ssh = new SSH2($this->server->getIp(), 22);
+        $ssh->login("pure", $serverPassword);
+        $ssh->setTimeout(360);
+
+        $rsyncCommand = "echo $serverPassword | sudo -S sudo apt-get install sshpass -y && sshpass -p '" . $this->ssh_root_password . "' ssh -o StrictHostKeyChecking=no " . $this->ssh_root_username . "@" . $this->ssh_host . " 'cd " . $this->location . " && zip -r - . | cat' > /home/pure/" . $this->backup_filename . ".zip";
```
Comment (fatihalp) [inline src/Jobs/BackupSite.php]: sudo apt-get install sshpass

buna neden gerek var? @spektra2147

File: src/Jobs/BackupDB.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    public function handle()
+    {
+        $serverPassword = $this->server->getPassword();
+
+        $ssh = new SSH2($this->server->getIp(), 22);
+        $ssh->login("pure", $serverPassword);
+        $ssh->setTimeout(360);
+
+        $rsyncCommand = "echo $serverPassword | sudo -S sudo apt-get install -y postgresql-client-14 && pg_dump -h " . $this->db_host . " -p " . $this->db_port . " -U " . $this->db_root_username . " -Fc " . $this->db_name . " > /home/pure/" . $this->backup_filename . ".sql";
```
Comment (fatihalp) [inline src/Jobs/BackupDB.php]: sudo apt-get install -y postgresql-client-14 ?

File: src/Jobs/BackupSite.php (hunk 1 above)
Comment (spektra2147) [reply src/Jobs/BackupSite.php]: removed

File: src/Jobs/BackupDB.php (hunk 2 above)
Comment (spektra2147) [reply src/Jobs/BackupDB.php]: removed

**Review by emrullahardc — APPROVED** [review]

**Pull request merged**